projects/plugins/jetpack/_inc/lib/class-jetpack-tweetstorm-helper.php

```diff
@@ -899,7 +916,7 @@ private static function get_boundary( $block, $offset ) {
 
 		$block_def = self::get_block_definition( $block['name'] );
 
-		if ( isset( $block_def['content'] ) && count( $block_def['content'] ) > 0 ) {
+		if ( isset( $block_def['content'] ) && is_countable( $block_def['content'] ) && count( $block_def['content'] ) > 0 ) {
```

Comment:
`$block_def` can only ever be null or come from an entry in the private `self::$supported_blocks`, which all have `'content'` either not set or set to an array. There's not even any filter that might add other kinds of values. So we should be safe to do like this (since `empty( array() )` returns true):
```suggestion
		if ( ! empty( $block_def['content'] ) ) {
```